app/forms/applicant_form.rb

```diff
@@ -14,7 +14,7 @@ class ApplicantForm
   validates :proceeding_type,
             presence: true,
             inclusion: { in: PROCEEDING_TYPES.values, allow_nil: true },
-            if: -> { level_of_help != "controlled" }
+            if: -> { !FeatureFlags.enabled?(:controlled) && level_of_help != "controlled" }
```

Comment:
I think this is the wrong feature flag. We're only introducing the `matter_type` screen behind the `asylum_and_immigration` feature flag, so if that flag is switched off, this is the only screen that certificated proceeding types can be added